yjit/src/asm/arm64/opnd.rs

```diff
@@ -0,0 +1,191 @@
+/// This operand represents a signed immediate value.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub struct Arm64Imm
```

Comment:
I think we should rename the opnds to A64Imm, A64Opnd just for conciseness, since we might be using them a lot.

But leave the module name as asm/arm64